keras/backend/theano_backend.py

```diff
@@ -862,11 +860,19 @@ def int_or_none(value):
     if filter_shape is not None:
         filter_shape = tuple(int_or_none(v) for v in filter_shape)
 
-    conv_out = T.nnet.conv2d(x, kernel,
-                             border_mode=th_border_mode,
-                             subsample=strides,
-                             input_shape=image_shape,
-                             filter_shape=filter_shape)
+    if filter_dilation == (1, 1):
```

Comment:
Safe choice! You can mark it with a deprecation warning or a TODO for us to remove it a few months later.